examples/Demo/Shared/Components/SiteSettingsPanel.razor.cs

```diff
@@ -9,6 +9,7 @@ public partial class SiteSettingsPanel
     private string? _status;
     private bool _popVisible;
     private bool _ltr = true;
+    private FluentDesignTheme? _theme;
```

Comment:
I don't know why you added a Ref, but ok.

Reply:
It is to call the new reset functionality (delete cache +local storage)